src/common/utils.js

```diff
@@ -0,0 +1,6 @@
+import { unicode } from "anvaad-js";
+
+const convertToUnicode = (word) => {
```

Comment:
This function has no error handling for invalid input or conversion failures. If word is null, undefined, or causes the unicode function to throw, the app might crash.

Reply:
Fixed.